ch1-lab-setup/vagrant/vagrantfile

```diff
@@ -1,23 +1,24 @@
 # This homelab consists of 4 linux VMs (2xCentOS + 2xUbuntu)
 Vagrant.configure("2") do |config|
-    n=2
+    n=1
```

Comment:
keep n=2 since we need to have 4 VMs

Reply:
yes, my laptop don't enough RAM to run 4 vm so i change it.